### PR TITLE
[#133559651] MySql findWhereEq

### PR DESCRIPTION
**Pull request by jonajonlee**

### CODE REVIEW

**Review by mattsmee — APPROVED** [review]
:shipit:

File: test/lib/mysql/index.spec.js [hunk 1]
```diff
@@ -105,6 +105,108 @@ describe('lib/mysql', function() {
...
+  describe('::findWhereEq', function() {
+
+    it('should execute a select query on the given table', function() {
+
+      const table = 'post_archive'
+
+      const request = {
+        projection : ['id', 'traversal_id', 'network_type_code']
+      , predicates : { stored_es : 1479439176, deleted : 0 }
+      , sort       : { id : 'desc' }
```
Comment (jxbadam) [inline test/lib/mysql/index.spec.js]: If sort is an object this is bad.

How do we do multiple sorts (for instance last_name DESC, first_name DESC, id ASC)? 
Key ordering in objects is not guaranteed. We may need to use a tuple.

**Review by jxbadam — CHANGES_REQUESTED** [review]

File: test/lib/mysql/index.spec.js (hunk 1 above)
Comment (scull7) [reply test/lib/mysql/index.spec.js]: I'm sorry I missed that @jonajonlee !  We should be using a tuple like so:

``` javascript
const query = {
  projection: [ 'id', 'traversal_id', 'network_type_code' ]
, predicates : { stored_es : 1479439176, deleted : 0 }
, sort: [ {"id": "desc"} ]
}

```

Comment (jonajonlee) [reply test/lib/mysql/index.spec.js]: @jxbadam @scull7 fixed. please review

**Review by scull7 — APPROVED** [review]

**Pull request merged**